frameworks/Java/tapestry/install.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-fw_depends java7 resin maven
+fw_depends java8 resin-java8 maven
```

Comment:
Agreed. If we have to lock down the java version prior to installing resin, it would be better to just remove the line `fw_depends java7` from `resin.sh` and rely on the caller explicitly setting the java version desired via `fw_depends java7 resin` or `fw_depends java8 resin`, instead of creating `resin-java8.sh`. Perhaps we could replace `fw_depends java7` with a check that either `java7.installed` or `java8.installed` exists and fail if that is not true just as one sanity check